Tasks/DockerV2/dockerpush.ts

```diff
@@ -79,12 +82,14 @@ export function run(connection: ContainerConnection, outputUpdate: (data: string
     let output = "";
     let outputImageName = "";
     let digest = "";
+    let imageSize = "";
     let promise = pushMultipleImages(connection, imageNames, tags, commandArguments, (image, commandOutput) => {
         output += commandOutput;
         outputImageName = image;
-        digest = extractDigestFromOutput(commandOutput);
-        tl.debug("outputImageName: " + outputImageName + "\n" + "commandOutput: " + commandOutput + "\n" + "digest:" + digest);
-        publishToImageMetadataStore(connection, outputImageName, tags, digest, dockerFile).then((result) => {
+        digest = extractFromOutput(commandOutput, matchPatternForDigest);
+        imageSize = extractFromOutput(commandOutput, matchPatternForSize);
+        tl.debug("outputImageName: " + outputImageName + "\n" + "commandOutput: " + commandOutput + "\n" + "digest:" + digest + "imageSize:" + imageSize);
+        publishToImageMetadataStore(connection, outputImageName, tags, digest, dockerFile, imageSize).then((result) => {
```

Comment:
We should ignore all failures silently when publishing to imagestore and trace it without affecting task status

Reply:
Done